CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -104,15 +104,33 @@ public bool TryCallMethod(long objectId, string name, object[] parameters, out o
 
             try
             {
-                // Do we have enough arguments? Add Type.Missing for any that we don't have incase of optional params
+                //Fix for C# paramArray and refactor on arguments mismatch 03/22/2016
```

Comment:
Again, not sure this comment is useful, seems like something you'd add to your own project, not so much an open source one? 

If your keen on referencing an `issue`, or `pull request`, you'd either open an issue to reference this or add a subsiquent commit one the `pull request` has been opened, like `Add paramArray support #1644`.

By all means add more comments, even comment what each branch of the if statement does.